docs/root/configuration/network_filters/redis_proxy_filter.rst

```diff
@@ -48,7 +48,8 @@ Per command statistics
 ----------------------
 
 The Redis filter will gather statistics for commands in the
-*redis.<stat_prefix>.command.<command>.* namespace.
+*redis.<stat_prefix>.command.<command>.* namespace. By default latency stats are in milliseconds and can be
+changed to microseconds by providing the configuration parameter :ref:`latency_in_micros <envoy_api_msg_config.filter.network.redis_proxy.v2.RedisProxy>` as true.
```

Comment:
Use the field ref (`envoy_api_field_config.filter.network.redis_proxy.v2.RedisProxy.latency_in_micros`) instead of the message ref.